deploy/charts/emqx-operator/values.yaml

```diff
@@ -39,19 +39,21 @@ tolerations: []
 affinity: {}
 
 admissionWebhooks:
+  failurePolicy: Fail
```

Comment:
What is `failurePolicy` ? Does this appear to be `certgen` specific ? Maybe it should in `admissionWebhooks.certgen.path.$someField`